UefiTestingPkg/AuditTests/PagingAudit/UEFI/AArch64/PagingAuditProcessor.c

```diff
@@ -268,13 +267,17 @@ DumpPlatforminfo (
     ElString = "Unknown";
   }
 
+  MemoryAttributeProtocol = NULL;
+  gBS->LocateProtocol (&gEfiMemoryAttributeProtocolGuid, NULL, (VOID **)&MemoryAttributeProtocol);

```

Comment:
I think this should check that `EFI_NOT_FOUND` is not returned in addition to checking whether the point is non-null to consider finding the attribute successful.

Reply:
Updated to set MemoryAttributeProtocol to NULL if the locate protocol call returns an error.